scripts/main.py

```diff
@@ -334,107 +334,116 @@ def start_interaction_loop(self):
         # Interaction Loop
         loop_count = 0
         while True:
-             # Discontinue if continuous limit is reached
+            # Discontinue if continuous limit is reached
```

Comment:
Revert this whitespace fix unless requested by the linter. We fix whitespace in separate PRs once a day to reduce breakage. And the flake8 config is continuously tightened, but incrementally, to avoid massive breakage of PRs.